manage_db.py

```diff
@@ -16,7 +16,9 @@ def myInput(text):
 
         menu = myInput('Selection:')
         if menu == 'i':
-            Database.buildIndex()
+            print('Suggested networks: BLOOMBERG, FBC, CNBC')
+            network = myInput('Network?:')
+            Database.buildIndex(network)
```

Comment:
Can this be a multiple choice with an option for custom input? Simple spelling errors may cause unintended effects.
Also I know Archive has a category for 'TV-FOXNEWS'. I'm not sure if FBC is a subset of that or a separate show.

Reply:
FBC is the Fox Business Channel. It may be a subset or something separate, but in any case we don't want to get all of the non-finance shows from Fox News.